frontend/src/SearchBar/SearchBar.jsx

```diff
@@ -1,61 +1,80 @@
 import { useState, useEffect } from 'react';
+import { CiSearch } from "react-icons/ci";
 import "./SearchBar.css"
+import { useNavigate } from 'react-router';
 
-function SearchBar (){
-    const [search, setSearch] = useState('');
-    const [recipes, setRecipes] = useState([]);
-    const [url, setUrl] = useState()
-    const BASE_URL = `https://api.spoonacular.com/recipes/complexSearch`;
+function SearchBar (props){
+    const [input, setInput] = useState('');
+    const navigate = useNavigate()
 
-    useEffect(() => {
-        if (search.trim() === '') {
-            setRecipes([]);
-            return;
-        }
 
-        const fetchRecipes = async () => {
-            try {
-                const response = await fetch(`${BASE_URL}?q=${search}&apiKey=${import.meta.env.VITE_API_KEY}`);
-                if (!response.ok) {
-                    throw new Error('Network response was not ok');
-                }
-                const data = await response.json();
-                setRecipes(data.results);
-            } catch (error) {
-                console.error('Error fetching data:', error);
-            }
-        };
+    const submitHandler = (e) =>{
+        e.preventDefault();
+        navigate('/searched/'+input);
```

Comment:
you can use template literals in here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
so it'll be `/searched/${input}`, you would need to wrap in with quotes like ``